docs/pywbemcligeneraloptions.rst

```diff
@@ -92,8 +90,7 @@ information on each command-group arguments and options):
   can be used to execute pywbemcli commands without access to a real server.
   See chapter :ref:`Mock WBEM server support` for information on defining
   mock servers.
-* **--log/-l** - See:ref:`Pywbemcli defined logging`. see
-  :ref:`Pywbemcli defined logging`
+* **--log/-l** - See:ref:`Pywbemcli defined logging`.
 * **--verbose/-v** -  Display extra information about the processing.
 * **--version** - Show the version of this command and of the pywbem package
       imported then exit.
```

Comment:
Second line breaks in the paragraph flow. Outdent it.

Reply:
DONE

Reply:
I need to review the doc output before marking this one resolved